lib/Support/regcomp.c

```diff
@@ -50,7 +50,7 @@
 #include "regcname.h"
 
 #include "llvm/Config/config.h"
-#include "llvm/Support/Compiler.h" // HLSL Change - for LLVM_C_FALLTHROUGH
+#include "llvm/Support/Compiler.h" // HLSL Change - for LLVM_FALLTHROUGH
```

Comment:
Hang on. If this is a C file, shouldn't it use LLVM_C_FALLTHROUGH?

Reply:
Fixed.